src/encoded/tests/test_audit_file.py

```diff
@@ -249,6 +249,40 @@ def test_audit_file_mismatched_paired_with(testapp, file1, file4):
                'inconsistent paired_with' for error in errors_list)
 
 
+def test_audit_paired_with_non_fastq(testapp, file1, file6):
+    testapp.patch_json(file1['@id'], {
+                        'run_type': 'paired-ended',
+                        'paired_end': '1'})
+    testapp.patch_json(file6['@id'], {
+                        'run_type': 'paired-ended',
+                        'platform': platform1['uuid'],
+                        'paired_end': '2',
+                        'paired_with': file1['uuid']})
+    res = testapp.get(file1['@id'] + '@@index-data')
+    errors = res.json['audit']
+    errors_list = []
+    for error_type in errors:
+        errors_list.extend(errors[error_type])
+    assert any(error['category'] ==
+               'paired with non-fastq' for error in errors_list)
+
+def test_audit_paired_with_fastq(testapp, file1, file4):
+    testapp.patch_json(file1['@id'], {
+                        'run_type': 'paired-ended', # @pytest.fixture defined file1 as a single-end. Changing it to paired-end
```

Comment:
In general don't add comments inside function argument definitions.  If they need explanation declare/describe them above where they are use.  Also comments at the the end of lines just makes them longer.  Put them above the line instead.

There are many ways to accomplish the goal of the comment however, 
- you could highlight the change by just making it a variable with a good name like
```run_type_override = 'paired-ended'```
- put the comment above the function declaration
- or a combination of both.

I prefer the first method because the code describes what is happening.  If you find yourself needing to add a comment, first check if the code could be made clearer.

Reply:
removed the comment